docs/content/preview/develop/build-apps/cloud-add-ip.md

```diff
@@ -18,6 +18,7 @@ In addition, **if you are using YugabyteDB Managed**, you need the following to
 
```

Comment:
In addition, **if you are using YugabyteDB Managed**,

Why is this in bold? There's already "Using a cluster in YugabyteDB Managed" at the top